docs/crocks/Either.md

```diff
@@ -0,0 +1,89 @@
+# Either
+
+`Either a b`
```

Comment:
Would be nice to go with the `Either c a` for 🌽sistancy now that I see it in play.